composer.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"require": {
 		"elasticsearch/elasticsearch": "~1.3",
-		"illuminate/support": "~4.2"
+		"illuminate/support": "~5.0"
```

Comment:
Better to use `^4|~5` to still support 4.2 of the framework see http://semver.mwl.be/#?package=illuminate%2Fsupport&version=%5E4%7C~5&minimum-stability=stable